docs/smart-contracts/creator-tools/Selling1155.mdx

```diff
@@ -64,6 +64,9 @@ Royalties are set on the main 1155 contract.
 - `royaltyBPS`: The royalty amount in basis points for secondary sales.
 - `royaltyRecipient`: The address that will receive the royalty payments.
 
+`*Note*: The `MintSchedule` argument has been deprecated.`
```

Comment:
should be *Note*: The `royaltyMintSchedule` argument has been deprecated.

Reply:
done!